### PR TITLE
Update list of support locales

### PR DESCRIPTION
**Pull request by flodolo**
Removed locales (no builds): ak, be, csb, ku, lg, mn, nso, oc, sah, sw, ta-LK, wo
Added locales: cak, kab, uz

### CODE REVIEW

File: js/locale/date_format.json [hunk 1]
```diff
@@ -41,12 +38,12 @@
     "ca": {
         "monthNameShort": "gen,febr,març,abr,maig,juny,jul,ag,set,oct,nov,des"
     },
+    "cak": {
+        "monthNameShort": "nab'ey ik',ruka'n ik',Mar,Rukaj ik',ro' ik',Ruwaq ik',Ruwuq ik',ruwaqxaq ik',rub'elej ik',rulaj ik',rujulaj ik',rukab'laj ik'"
```
Comment (georgf) [inline js/locale/date_format.json]: I have no good context here, but this doesn't look particularly short?

Comment (flodolo) [inline js/locale/date_format.json]: I've imported the strings, but they were too long, so I reverted back to English in the third commit.

Let me know if you prefer to have them squashed, or if you can squash them on merge.

File: js/fhr_v4.js [hunk 2]
```diff
@@ -87,6 +87,12 @@ function drawGraph(startupTimes) {
         // We need to localize our month names so first load our localized data,
         // then set the graph options and draw the graph.
         $.getJSON('js/locale/date_format.json', function(data) {
+            // Get localized month names, or fallback to en-US if they're not
+            // available
+            let localizedMonthNames = (currentLocale in data)
+                ? data[currentLocale].monthNameShort.split(',')
+                : data['en-US'].monthNameShort.split(',');
```
Comment (georgf) [inline js/fhr_v4.js]: Nit - no need to repeat the split() part:
```
let locale = ... ? data[currentLocale] : data['en-US'];
let localizedMonthNames = locale.monthNameShort.split(',');
```

**Comment by georgf** [comment]
Is there testing planned after landing this?

**Comment by flodolo** [comment]
Thanks, comment addressed (I used `localeData` for the variable name).

I've tested this locally, and I have a pending bug for Uzbek that needs testing on stage. Once merged, I'll run a test for a few locales against stage to make sure everything looks good.

When everything looks good, I will need a bug to update production, since we also have other localization updates pending.

**Pull request merged**